views/movies/edit-movie.hbs

```diff
@@ -0,0 +1,36 @@
+<h2>Edit the movie</h2>
+
+<form action="/movies/{{this._id}}/edit" method="post">
+
+
+    <label> Title:
+        <input type="text" name="title" value="{{this._id.title}}" />
+    </label>
```

Comment:
One clue is here, you want to print title - correct. Why are you accessing _id then as well ? ;)